language/move-lang/src/unit_test/filter_test_members.rs

```diff
@@ -103,9 +103,12 @@ fn check_has_unit_test_module(context: &mut Context, prog: &P::Program) -> bool
 fn filter_tests_from_definition(
     context: &mut Context,
     def: P::Definition,
+    is_lib: bool,
```

Comment:
nit, the rest of the compiler tends to use `is_source_def` or something similar. Might be best to keep that